gradle/publish-bintray.gradle

```diff
@@ -78,33 +77,23 @@ publishing {
             case 'kotlinMultiplatform':
                 // With Kotlin 1.4 & HMPP, the root module should have no suffix in the ID, but for compatibility with
                 // the consumers who can't read Gradle module metadata, we publish the JVM artifacts in it, too
-                it.artifactId = isKotlin137x ? "$project.name-native" : project.name
-                if (!isKotlin137x) {
-                    apply from: "$rootDir/gradle/publish-mpp-root-module-in-platform.gradle"
-                    publishPlatformArtifactsInRootModule(publications["jvm"])
-                }
+                it.artifactId =  project.name
+                apply from: "$rootDir/gradle/publish-mpp-root-module-in-platform.gradle"
+                publishPlatformArtifactsInRootModule(publications["jvm"])
                 break
             case 'metadata':
                 // As the old -common dependencies will fail to resolve with Gradle module metadata, rename the module
                 // to '*-metadata' so that the resolution failure are more clear
-                it.artifactId = isKotlin137x ? "$project.name-common" : "$project.name-metadata"
+                it.artifactId = "$project.name-metadata"
                 break
             case 'jvm':
-                it.artifactId = isKotlin137x ? project.name : "$project.name-jvm"
+                it.artifactId = "$project.name-jvm"
                 break
             case 'js':
             case 'native':
                 it.artifactId = "$project.name-$type"
                 break
```

Comment:
seems now, all those switch cases are the same :)

Reply:
thanks!